src/renderer/dx/DxRenderer.cpp

```diff
@@ -979,10 +1011,10 @@ try
 
             // Now let go of a few of the device resources that get in the way of resizing buffers in the swap chain
             _dxgiSurface.Reset();
-            _d2dRenderTarget.Reset();
+            _d2dDeviceContext.Reset();

```

Comment:
Do we want to keep the D2D device context on resize? If so, something like this would probably work:
```c++
_dxgiSurface.Reset();
_d2dBitmap.Reset();
_d2dDeviceContext->SetTarget(nullptr);
```

Reply:
Good call. I just naturally replaced the render targets with device contexts... but you're right. I bet the context is still fine, it's just those pieces that are adjusting (and they were all hidden/packed inside the same render target before.)